sustainml_docs/rst/formalia/titlepage.rst

```diff
@@ -0,0 +1,27 @@
+.. raw:: html
+
+  <h1>
+    eProsima SustainML Documentation
+  </h1>
+
+.. image:: /rst/figures/logo.png
+  :height: 100px
+  :width: 100px
+  :align: left
+  :alt: eProsima
+  :target: http://www.eprosima.com/
+
+Application Aware, Life-Cycle Oriented Model-Hardware Co-Design Framework for Sustainable, Energy Efficient ML
+Systems project.
+*SustainML* is a framework and an associated toolkit that will foster energy efficiency throughout the whole
+life-cycle of ML applications: from the design and exploration phase that includes exploratory iterations of training,
+testing and optimizing different system versions through the final training of the production systems (which often
+involves huge amounts of data, computation and epochs) and (where appropriate) continuous online re-training during
+deployment for the inference process.
+The framework will optimize the ML solutions based on the application tasks, across levels from hardware to model
+architecture.
+It will also collect both previously scattered efficiency-oriented research, as well as novel Green-AI methods.
+
+AI developers from all experience levels can make use of the framework through its emphasis on human-centric
+interactive transparent design and functional knowledge cores, instead of the common blackbox and fully automated
+optimization approaches in AutoML.
```

Comment:
NIT: It would be nice adding a little EU reference (including the logo)

----

.. image:: /rst/figures/eu_flag.jpg
  :height: 45px
  :align: left

This project has received funding from the European Union’s Horizon Europe research and innovation programme (HORIZON-CL4-2021-HUMAN-01) under grant agreement No 101070408.

Reply:
Nice one!